tests/succeed/distillation/fresh-names.snap

```diff
@@ -0,0 +1,12 @@
+stdout = '''
+def a : () : Type = ();
```

Comment:
Looks like the type annotation on this item is being distilled in check mode instead of synth mode?

Reply:
Oooh, good catch